bot/cogs/Student-Teacher/README.md

```diff
@@ -0,0 +1,2 @@
+# Student-Teacher
+A reddit bot designed to inform users whether or not a Professor/Class is via Clemson's Grade Distribution Report data.
```

Comment:
This readme should be in a pydoc comment decorating the class declaration. That allows the help command to dynamically read the pydoc and show it to the user when the !help command is run